test/scenarios/el92-src@rpm-install.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+export SKIP_GREENBOOT=true
+export TEST_RANDOMIZATION=none
+
+scenario_create_vms() {
+    prepare_kickstart host1 kickstart-liveimg.ks.template ""
+    launch_vm host1 "rhel-9.2"
```

Comment:
"rhel-9.2" is the [default](https://github.com/openshift/microshift/blob/main/test/bin/scenario.sh#L12) blueprint name, do we need to be explicit about it ?

Reply:
I like being explicit, but it's not necessary.